src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlySame.java

```diff
@@ -29,4 +29,16 @@ public void forLoop() {
         for (; ; ) {
         }
     }
+
+    public @interface TestAnnotation {
+    }
+
+    public @interface TestAnnotation1 {String value();}
+
+    public @interface TestAnnotation2 {
+        String someValue();
+    }
+
+    public @interface TestAnnotation3 {String value();
+    }
```

Comment:
Same can never produce a violation?
How come we have one in regression? https://sd1998.github.io/checkstyle-regression/Fix-7041/checkstyle/index.html#A14

Reply:
@rnveach It can produce violation and I have added the case for the same in `InputRightCurlyLeft.java`. `InputRightCurlySame.java` is only being used in `testSameDoesNotComplainForNonMultilineConstructs()` and contains cases which should not raise a violation for `RightCurlyOption.SAME`. Thus I have not added the case where violations would be raised. But the violation case for `RightCurlyOption.SAME` has been added in `InputRightCurlyLeft` [here](https://github.com/checkstyle/checkstyle/blob/026b6f7f09dafcb10b8ea11ef6beb02485c9c70f/src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyLeft.java#L202) which can also be seen in the `testSame()` method [here](https://github.com/checkstyle/checkstyle/blob/026b6f7f09dafcb10b8ea11ef6beb02485c9c70f/src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheckTest.java#L80)

Reply:
Thanks. It was not clear from the diff.